docs/UIToolkit/Bundles/CIRAConfigs.md

```diff
@@ -1,91 +1,100 @@
-# Quickstart - To Bundle CIRA configs Control  
+# Quickstart - Bundle CIRA Config Control
 
-This document shows how to bundle CIRA configs control and display it on a sample html page for testing.
-
-  - how to customize the control
-  - how to create a bundle for CIRA configs control
-  - how to add to sample html
-  - how to add a new language for Internalization 
+This document shows how to:
 
+- Run the CIRA Config control in development enviroment
+- Create a bundle for CIRA Config control
+- Add bundle to a sample HTML file 
 
 ## Prerequisites
 
 In order to deploy and make changes, the following tools and application has to be installed on your development machine
--   [Git](https://git-scm.com/)
--   [Visual Studio Code](https://code.visualstudio.com/) or any other IDE 
--   [Node.js](https://nodejs.org/)
--   [RPS](https://github.com/open-amt-cloud-toolkit/rps) stands for **R**emote **P**rovisioning **S**erver
--   Build and deploy RPS MicroService locally
--	Intel AMT device is configured and connected to RPS. See the [RPS](https://github.com/open-amt-cloud-toolkit/rps) for documentation.
--   Chrome 
+
+- [Git](https://git-scm.com/)
+- [Visual Studio Code](https://code.visualstudio.com/) or any other IDE 
+- [Node.js](https://nodejs.org/)
+- [Chrome* Browser](https://www.google.com/chrome)
+- [A MPS and RPS Server with an AMT Device Connected](../../../Docker/dockerLocal/)
+
 
 ## Download and Install UI Toolkit
 
-At a command prompt, run the following commands:
-```
-git clone https://github.com/open-amt-cloud-toolkit/ui-toolkit
-cd ui-toolkit
-npm install
-```
+1. Open a Terminal (Linux) or Command Prompt (Windows) and navigate to a directory of your choice for development.
+
+2. Clone the UI Toolkit Repository.
+	```
+	git clone https://github.com/open-amt-cloud-toolkit/ui-toolkit#v1.2.0
+	```
+
+3. Change to the `ui-toolkit` directory.
+	```
+	cd ui_toolkit
+	```
+
+4. Install the dependencies.
+	```
+	npm install
+	```
 
-## Customize the control
+## Run in Development Enviroment
 
-To add new changes and test the changes  instantly before bundling the control, webpack dev server can be used
+To add and test new changes before bundling the control, a webpack dev server can be used.
```

Comment:
To add and test new changes before bundling the control, use a webpack dev server.

Reply:
I know it sounds trivial, but the way it's written it sounds as if the web dev server is driving the action when it's the user.